docs/standard/serialization/snippets/system-text-json-how-to/csharp/RoundtripToString.cs

```diff
@@ -31,7 +31,7 @@ public static void Run()
             Console.WriteLine($"Pretty-printed JSON output:\n{jsonString}\n");
 
             // <SnippetDeserialize>
-            weatherForecast = JsonSerializer.Deserialize<WeatherForecastWithPOCOs>(jsonString);
+            weatherForecast = JsonSerializer.Deserialize<WeatherForecast>(jsonString);
```

Comment:
I updated this because the doc read like this, which was confusing to me:

![image](https://user-images.githubusercontent.com/24882762/95642054-ef1e3200-0a5a-11eb-9cba-179462affad4.png)

So there's a mismatch between the class name, WeatherForecast, and the type in the generic type parameter (WeatherForecastWithPOCOs).